src/DurableTask.AzureStorage/Messaging/Session.cs

```diff
@@ -60,7 +54,7 @@ public void TraceProcessingMessage(MessageData data, bool isExtendedSession)
             }
 
             TaskMessage taskMessage = data.TaskMessage;
-            CloudQueueMessage queueMessage = data.OriginalQueueMessage;
+            CloudQueueMessage queueMessage = data.OriginalQueueMessage.CloudQueueMessage;
```

Comment:
Shouldn't this change be to switch from `CloudQueueMessage` to `QueueMessage`?
```suggestion
            QueueMessage queueMessage = data.OriginalQueueMessage;
```